corehq/toggles/__init__.py

```diff
@@ -756,6 +756,13 @@ def _ensure_valid_randomness(randomness):
     [NAMESPACE_DOMAIN]
 )
 
+SSO_REMOTE_USER_MANAGEMENT = StaticToggle(
+    'sso_remote_user_management',
+    "Shows remote user management fields in SSO Identity Provider Form",
+    TAG_PRODUCT,
+    [NAMESPACE_DOMAIN, NAMESPACE_USER]
```

Comment:
I haven't seen this syntax before. Is `NAMESPACE_USER` necessary? It looks like we only call this toggle via `enabled_for_request()`, and that function only seems to work with `NAMESPACE_DOMAIN`, `NAMESPACE_EMAIL_DOMAIN`, or `None`

Reply:
This allows the feature flag can be either enabled for domain or enabled for a certain user. I tested it locally, both scenario works.

Reply:
This was my oversight. Oddly, it looks like we translate `NAMESPACE_USER` into `None`, so that's why this toggle works for User access. I'm curious why we'd support both, though. I'd imagine we have "SSO Admins" that span domains, so we want to support user access, but why do we also support domains?

Reply:
@jingcheng16 best practice is to create separate PR just with the commit adding this toggle. That is because this file is changed quite often and has the potential for merge conflicts as time goes on. Can you please do this?

@mjriley the implementation here is fine :)